auth-web/src/components/auth/manage-business/AuthorizationEmailSent.vue

```diff
@@ -25,19 +22,30 @@
     </v-card>
 </template>
 <script lang="ts">
-import { defineComponent } from '@vue/composition-api'
+import { computed, defineComponent } from '@vue/composition-api'
 
 export default defineComponent({
+  name: 'AuthorizationEmailSent',
+  props: {
+    email: {
+      type: String,
+      default: ''
+    }
+  },
   setup (props, { emit }) {
     const openHelp = () => {
       emit('open-help')
     }
     const closeAuthEmailSentDialog = () => {
       emit('close-dialog')
     }
+    const propsEmail = computed(() => {
```

Comment:
is this necessary? props are usually passed through. If you're binding to `v-model` it makes sense to do this though.